Settings.md

```diff
@@ -0,0 +1,52 @@
+Show Action Notifications: If you want to see messages relating to your current action.
+Show Combat Drops: If you want to see drops from combat.
+Show Combat Timers/Warnings: If you want to see Timers and Warnings from Combat. (like enrage)
+Show Combat Experience: If you wish to see experienced earned from monsters in combat on kill.
+Show Rare Drops: If you want to see any Rare drops you've received.
+Show Flavor Messages: If you want to see special messages from certain zones and actions.
+Show Enchantment Procs: If you want to see procs from your Enchantments (Like Inferno).
+Show Shard Messages: If you want to see when you earned a shard from skilling.
+<b> Sound </b>
+Sound on Mention: If a sound plays when your name is mentioned in chat.
+Sound on Whisper: If a sound plays when a player whispers you.
+Sound on Group Invite: If a sound plays when an invite is received.
+Volume: How loud you want it to be.
+<b> Combat </b>
+Auto Eat: What threshhold of percent HP you wish to automatically eat food to heal yourself at.
+Max Difficulty: What you wish to set your Global Max Difficulty at.
+Starting Difficulty: What you wish to set your Global Starting Difficulty at.
+Force Full Inventory: If you want your food inventory to be full before it starts combat. Will not let you start combat if its not full if active.
+Show Inventory while Grouped: If you want to see your inventory while grouped with others.
+Allow Invites from: Select who is allowed to invite you.
+Automatic Priority Rotation: If you want the game to automatically set your combat skill rotation.
+Force Valid Merc Loadout: If you want the game to make sure your merc's are properly equipped before combat. Will not start combat if invalid if active.
+Dungeon Misclick Safety: If you want to prevent accidently leaving dungeons.
+Ignore Dungeon Update Safety: ?
+Target Threat Bars: Your combat targeting priority. Does not fully work as intended.
+<b> Tooltips </b>
+Modern Style: A more modern style of tooltip for your items. Turn off to use oldschool Tooltips
+Compact Tooltips: Weither to hide overly bloated information. Will reduce tooltip size.
+Equipped Comparison: Weither to show your currently equipped items when you hover over an item that goes into that equip slot.
+<b> Privacy </b>
+Anonymous: If you want your name to be displayed or not in Community Data Releases.
+<b> Misc </b>
+Auto Vault Like Items: If you want items you obtained to automatically be put in the vault if they are already in your vault.
+Auto Tacklebox Like Items: If you want bait to automatically be put in your tacklebox if they are already in there.
+Use Legacy Sorting: If you want to sort items the way they used to be sorted. (This will ignore other settings)
+Sort by Value before ID: If you want to sort items by value instead of their item ID's
+High Performance Mode: Will disable some animations and graphics to prioritize Performance.
+Enable Pinned Tabs: If you want to pin tabs for quicker access.
```

Comment:
Would prefer if we have a more general description here. Otherwise it will be outdated in a week :D

Reply:
Fair

Reply:
I'll move the setting info to a separate note file just in case.